iphone/Maps/UI/Settings/Cells/SettingsTableViewLinkCell.swift

```diff
@@ -2,18 +2,25 @@
 final class SettingsTableViewLinkCell: MWMTableViewCell {
   @IBOutlet fileprivate weak var title: UILabel!
   @IBOutlet fileprivate weak var info: UILabel?
+  
+  private let chevronImageView: UIImageView = {
+    let imageView = UIImageView(image: UIImage(named: "ic_arrow_gray_right"))
+    imageView.size = CGSize(width: 11, height: 28)
```

Comment:
How these sizes were calculated?

Reply:
The size of disclosure indicator that is used by default in table view's cells when selected accessory type is 11x14pt (i have checked in different ios version and screen sizes), but the problem with tis indicator that you jest cannon change its tint color (well know problem in ios), and you should use your own accory Image.
So i found that same already used in assets - ic_arrow_gray_right, but it needs to be resized to have same look as default. Not best approach but there is no other (except implementing of using system theme dark/light modes (from ios13) that is automatically handle image tint color)

Reply:
1. Adding this comment into the code would be beneficial for future readers ;-)
2. Can we use [this SVG](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aarrow_forward_ios%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4024) or [any other SVG](https://fonts.google.com/icons?icon.query=right+&icon.platform=web) instead of resizing a pixel image?
3. Can we reuse iOS 13 theme features on iOS 13+ devices conditionally? Or would it be significantly simpler/easier to drop iOS 12?

Reply:
1. thanks, got it!
2. yes! I tried not to add new assets, but if it ok - i'll add apples chevron default svg from apples SFSymblols
3. because of we are using custom Style system (with MWMThemeManager, MWMSettings, MWMRenderers and so on) when we tap on changing theme cell is Settings we are **actually do not change** UIKit's native (from ios13)`traitCollection.userInterfaceStyle` ... UIKit theme system work with same approach: goes through views hierarchy and change views colors. So both systems do the same work and mix them may cause unexpected behaviour... And i really don't find the way to use both systems for some views: whole design system looks very complex and hard to debug now...

Reply:
If the visual difference is negligible, the existing icon is also ok. But otherwise, you can import/use any better icons (just make sure that they work on iOS 12).

Regarding the dark theme, let's create another issue (or find an appropriate existing one) to discuss how to refactor it. For example, would it be very easy for us to support if we implement only the system theme support starting from iOS 13, and leave iOS 12 users without the dark theme? What are the other pros and cons of dropping iOS 12 completely?